test/test_ops.py

```diff
@@ -97,6 +98,22 @@
         op_db,
     )
 )
+# This list specifies the ops that should be skipped by `test_numpy_ref_mps`. Currently a large chunk of
+# `op_db`'s tests fail, but as MPS is brought in line with numpy, this list should disappear.
+_ops_failing_on_mps = [
```

Comment:
OpInfos have a `skips` attribute and a decorator to skip the test can be added there. Adding the skips to the OpInfos keeps all the test data related to a particular operator in one place.

Reply:
d'oh, i even found that and then didn't think to use it for this. Done in https://github.com/pytorch/pytorch/pull/87342/commits/a1dc7a862088ef5878375bd3122216f4c694dae9